providers/trino/src/airflow/providers/trino/hooks/trino.py

```diff
@@ -92,6 +92,42 @@ class TrinoHook(DbApiHook):
     query_id = ""
     _test_connection_sql = "select 1"
 
+    @classmethod
+    def get_ui_field_behaviour(cls) -> dict[str, Any]:
+        """Return custom field behaviour."""
+        import json
```

Comment:
Why import here? It can be at top of the file.

Reply:
I accidentally wrote this incorrectly, and I removed it